rainier-cats/src/main/scala/com/stripe/rainier/cats/package.scala

```diff
@@ -65,3 +103,35 @@ private[cats] class MonadRandomVariable extends Monad[RandomVariable] {
       case Right(b) => RandomVariable(b)
     }
 }
+
+private[cats] trait EqInstances {
```

Comment:
I moved these into `rainier-cats` so that I could get access to them when writing my own tests outside the repo.